tasks/main.yml

```diff
@@ -1,4 +1,35 @@
 ---
+- name: Update apt cache (Debian derivatives)
+  become: true
+  ansible.builtin.apt:
+    cache_valid_time: "{{ gxadmin_apt_cache_valid_time }}"
+  when: >-
+    ansible_os_family == "Debian"
+    and gxadmin_manage_dependencies
+    and gxadmin_manage_apt_cache
+
+- name: Ensure role and gxadmin dependencies are installed
+  become: true
+  ansible.builtin.package:
+    name:
+      - git
+      - make
+    state: present
+  when: gxadmin_manage_dependencies
+
+- name: Ensure role and gxadmin dependencies are installed (EL)
+  become: true
+  ansible.builtin.package:
+    name: postgresql
+    state: present
+  when: ansible_os_family == "RedHat" and gxadmin_manage_dependencies
+
+- name: Ensure role and gxadmin dependencies are installed (Debian derivatives)
+  become: true
+  ansible.builtin.package:
+    name: postgresql-client
+    state: present
+  when: ansible_os_family == "Debian" and gxadmin_manage_dependencies
```

Comment:
Please merge these and provide separate facts for debian package names + rhel package names.